clippy_utils/src/msrvs.rs

```diff
@@ -186,27 +186,67 @@ impl MsrvStack {
 }
 
 fn parse_attrs(sess: &Session, attrs: &[impl AttributeExt]) -> Option<RustcVersion> {
-    let mut msrv_attrs = attrs.iter().filter(|attr| attr.path_matches(&[sym::clippy, sym::msrv]));
-
-    if let Some(msrv_attr) = msrv_attrs.next() {
-        if let Some(duplicate) = msrv_attrs.next_back() {
-            sess.dcx()
-                .struct_span_err(duplicate.span(), "`clippy::msrv` is defined multiple times")
-                .with_span_note(msrv_attr.span(), "first definition found here")
-                .emit();
-        }
-
-        if let Some(msrv) = msrv_attr.value_str() {
-            if let Some(version) = parse_version(msrv) {
-                return Some(version);
+    let mut first_clippy_attr = None;
+    let mut clippy_msrv = None;
+    let mut cfg_version: Option<RustcVersion> = None;
```

Comment:
Nit, this looks strange to have only one ascribed variable.
```suggestion
    let mut cfg_version = None;
```